stops_test.go

```diff
@@ -32,13 +32,17 @@ func GetClient(t *testing.T) *messaging.Client {
 	if err != nil {
 		t.Error(err)
 	}
-	client.SetupPublishing(messaging.JobsExchange)
+	client.SetupPublishing(exchangeName())
 	go client.Listen()
 	return client
 }
 
 func messagingURI() string {
-	return "amqp://guest:guest@rabbit:5672/"
+	return "amqp://guest:guest@rabbit:5672/%2f"
```

Comment:
The default vhost is `/`. We should be defaulting to `/de`. So in this example, the `vhost` section of the URI should be `.../%2fde`.